File: js/ejer30.js
//validar acceso

usuario="admin"

contrasena = "1234"

u = prompt("ingrese su usuario:")
c = prompt("ingrese su contraseña:")



if (usuario == u && contrasena == c) {
    alert("Bienvenido al sistema"+u+", su rol es: administrador");
} else {
    alert("Usuario o contraseña incorrectos");
}

File: python/ejer30.py
#validar acceso

usuario="admin"

contraseña = "1234"

u = input("ingrese su usuario:")
c = input("ingrese su contraseña:")



if usuario == u and contraseña == c:
    print("Bienvenido al sistema", u, " su rol es: administrador")
else:
    print("Usuario o contraseña incorrectos")
